YANG/ccamp/l1csm/ietf-l1csm.yang

```diff
@@ -106,46 +106,30 @@ module ietf-l1csm {
     reference
       "MEF63: Subscriber Layer 1 Service Attributes";
 
-    container endpoint-1 {
+    container endpoints {
       description
-        "One end of UNI id's - string and id";
-      leaf endpoint-id {
-        type string;
-        mandatory true;
+        "The container for the list of the subscriber L1VC end 
+        points";
+      list endpoint {
+        key "endpoint-id";
+        min-elements 2;
+        max-elements 2;
         description
-          "Subscriber end point ID of one end";
-      }
-      leaf uni {
-        type leafref {
-          path "/l1-connectivity/access/unis/uni/uni-id";
-        }
-        mandatory true;
-        description
-          "This is one end of subscriber L1VC end point ID value = 
-           UNI-1";
-      }
-    }
-    container endpoint-2 {
-      description
-        "One end of UNI id's - string and id";
-      leaf endpoint-id {
-        type string;
-        must '. != ../../endpoint-1/uni' {
-          error-message
-            "The two end point IDs must not be equal to each other. ";
+          "The list of the two of the subscriber L1VC end points";
+        leaf endpoint-id {
+          type string;
+          mandatory true;
+          description
+            "The subscriber L1VC end point ID of one end";
         }
-        mandatory true;
-        description
-          "Subscriber end point ID of the other end";
-      }
-      leaf uni {
-        type leafref {
-          path "/l1-connectivity/access/unis/uni/uni-id";
+        leaf uni {
+          type leafref {
+            path "/l1-connectivity/access/unis/uni/uni-id";
+          }
+          mandatory true;
+          description
+            "This UNI supporting the subscriber L1VC end point.";
```

Comment:
OLD: 
This UNI supporting the subscriber L1VC end point
NEW
The UNI supporting the subscriber L1VC end point

Reply:
Ok

```suggestion
            "The subscriber L1VC end point ID";
```